cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,120 @@
 /// <reference types='cypress' />
+const { faker } = require('@faker-js/faker');
+
+const username = faker.internet.userName();
+const lastname = faker.name.lastName();
+const email = faker.internet.email();
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
```

Comment:
add empty lines between beforeEach and all tests

Reply:
+